docs/hub/webhooks-guide-discussion-bot.md

```diff
@@ -0,0 +1,112 @@
+# Webhook guide: build a Discussion bot based on BLOOM
+
+Here's a short guide on how to use HF Webhooks to build a bot that replies to Discussion comments on the Hub with a response generated by BLOOM (using the Inference API).
+
+## Create your webhook in your user profile
+
+First, create your webhook by going to https://huggingface.co/settings/webhooks
+
+- Input a few target repositories that your webhook will run on.
```

Comment:
Pardon me if I want to dumb it down but can you perhaps clarify what it will do in the repository exactly?

Reply:
will rephrase